SKILL.tsv

```diff
@@ -2584,460 +2584,460 @@ SKILL_20150317_002583	Kneeling Shot: Enhance
 SKILL_20150317_003010	Doom Spike: Enhance 2
-SKILL_20150317_003011	Damage of [Doom Spike] is enhanced to Cataphract circle 3.
-SKILL_20150317_003012	Zombify : Large Zombie
-SKILL_20150317_003013	Chance of creating large zombies when zombifying monsters increases by 1% per attribute level.
-SKILL_20150317_003014	Zombify : Wheelchair Zombie
-SKILL_20150317_003015	Chance of wheelchair zombie when creating zombies increases by 1% per attribute level
-SKILL_20150317_003016	Flare shot: Enhance 3
-SKILL_20150317_003017	Damage of [Flare Shot] is enhanced to Scout Circle 3.
+SKILL_20150317_003011	$Enhanced the damage of [Doom Spike] to Cataphract's 3rd Circle standards.
+SKILL_20150317_003012	$Zombify: Large Zombie
+SKILL_20150317_003013	$The chance of creating large zombies when zombifying large monsters increases by 1% per attribute level.
+SKILL_20150317_003014	$Zombify: Wheelchair Zombie
+SKILL_20150317_003015	$The chance of a wheelchair zombie appearing when creating zombies increases by 1% per attribute level.
+SKILL_20150317_003016	$Flare Shot: Enhance 3
+SKILL_20150317_003017	$Enhanced the damage of [Flare Shot] to Scout's 3rd Circle standards.
 SKILL_20150317_003018	Astral Body Explosion: Enhance 2
-SKILL_20150317_003019	Damage of [Astral Body Explosion] is enhanced to Sadhu Circle 2.
+SKILL_20150317_003019	$Enhanced the damage of [Astral Body Explosion] to Sadhu's 2nd Circle standards.
 SKILL_20150317_003020	Astral Body Explosion: Enhance 3
-SKILL_20150317_003021	Damage of [Astral Body Explosion] is enhanced to Sadhu Circle 3.
-SKILL_20150317_003022	Use One-handed sword
+SKILL_20150317_003021	$Enhanced the damage of [Astral Body Explosion] to Sadhu's 3rd Circle standards.
+SKILL_20150317_003022	$Use of One-handed Swords
 SKILL_20150317_003023	Can use [One-handed Sword] type weapons.
-SKILL_20150317_003024	Use Two-handed sword
+SKILL_20150317_003024	$Use of Two-handed Swords
 SKILL_20150317_003025	Can use [Two-handed sword] type weapons.
-SKILL_20150317_003026	Use of One-handed spear
+SKILL_20150317_003026	$Use of One-handed Spears
 SKILL_20150317_003027	Can use [One-handed spear] type weapons.
-SKILL_20150317_003028	Use Two-handed spear
+SKILL_20150317_003028	$Use of Two-handed Spears
 SKILL_20150317_003029	Can use [Two-handed Spear] type weapons.
-SKILL_20150317_003030	Use Bow
+SKILL_20150317_003030	$Use of Bows
 SKILL_20150317_003031	Can use [Bow] type weapons.
-SKILL_20150317_003032	Use Crossbow
+SKILL_20150317_003032	$Use of Crossbows
 SKILL_20150317_003033	Can use [Crossbow] type weapons.
-SKILL_20150317_003034	Use Rod
+SKILL_20150317_003034	$Use of Rods
 SKILL_20150317_003035	Can use [Rod] type weapons.
-SKILL_20150317_003036	Use Staff
+SKILL_20150317_003036	$Use of Staves
 SKILL_20150317_003037	Can use [Staff] type weapons.
-SKILL_20150317_003038	Use Mace
+SKILL_20150317_003038	$Use of Maces
 SKILL_20150317_003039	Can use [Mace] type weapons.
-SKILL_20150317_003040	Use wand
+SKILL_20150317_003040	$Use of Wands
```

Comment:
[3022-3040] The "of" seem off in those corrections (pun intended)
I liked better the other translation, but how about changing it to "Enable [Weapon type]", and on the description "Enables to use [Weapon type] type weapons."?